.travis.yml

```diff
@@ -1,47 +1,34 @@
 language: php
-
-addons:
-  apt:
-    packages:
-      - tidy
-
-php:
-  - 5.6
-  - 7.0
-  - 7.1
-
-before_install:
-  - pip install --user codecov
-
-env:
-  global:
-    - DB=MYSQL CORE_RELEASE=4
+dist: precise
```

Comment:
For SS4 we should definitely be trying to use trusty if possible since precise is being phased out